### PR TITLE
Removing a useless instruction and generating XML comments

### PR DESCRIPTION
**Pull request by jmprieur**
- Removing a useless instruction (as pointed to by @jg11jg in https://github.com/AzureAD/microsoft-identity-web/issues/154)
- Regenerating the XML comments.
- Fixing a couple of static analysis warnings

### CODE REVIEW

File: src/Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs [hunk 1]
```diff
@@ -62,8 +62,8 @@ public IActionResult SignOut([FromRoute] string scheme)
                  {
                      RedirectUri = callbackUrl,
                  },
-                CookieAuthenticationDefaults.AuthenticationScheme,
-                scheme);
+                 CookieAuthenticationDefaults.AuthenticationScheme,
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs:65]: Addressed a warning about indentation

File: src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs [hunk 2]
```diff
@@ -40,8 +40,8 @@ public class MicrosoftIdentityOptions : OpenIdConnectOptions
         public string RedirectUri { get; set; }
 
         /// <summary>
-        /// In a web app, gets or sets the PostLogoutRedirectUri 
-        /// This property is exclusive with <see cref="RemoteAuthenticationOptions.SignedOutCallbackPath"/> which should be used preferably if you don't want
+        /// In a web app, gets or sets the PostLogoutRedirectUri
+        /// This property is exclusive with <see cref="OpenIdConnectOptions.SignedOutCallbackPath"/> which should be used preferably if you don't want
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs:44]: Addressed an XML comment warning (wrong property)

File: src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs [hunk 3]
```diff
@@ -129,10 +129,9 @@ public static class WebApiServiceCollectionExtensions
 
                 options.Events.OnTokenValidated = async context =>
                 {
-                    await onTokenValidatedHandler(context).ConfigureAwait(false);                
+                    await onTokenValidatedHandler(context).ConfigureAwait(false);
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs:132]: Removed extra spaces

File: src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs [hunk 4]
```diff
@@ -129,10 +129,9 @@ public static class WebApiServiceCollectionExtensions
...
                     context.HttpContext.StoreTokenUsedToCallWebAPI(context.SecurityToken as JwtSecurityToken);
                     context.Success();
-                    await Task.FromResult(0).ConfigureAwait(false);
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs:135]: Removed this line which is no longer necessary given we now call `await onTokenValidatedHandler(context).ConfigureAwait(false);`
Thanks @jg11jg for the heads-up in your comments for #154

File: src/Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs (hunk 1 above)
Comment (bgavrilMS) [reply src/Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs:65]: Don't worry about explaining these :). I sometimes use Code Maid to apply spacing rules across all open files. https://marketplace.visualstudio.com/items?itemName=SteveCadwallader.CodeMaid

**Pull request merged**